modules/webapp/webservers-compute.tf

```diff
@@ -56,8 +57,10 @@ resource "google_compute_instance_template" "webserver_template" {
   can_ip_forward = false
 
   scheduling {
-    automatic_restart = true
-    on_host_maintenance = "MIGRATE"
+    automatic_restart = !var.vm_flag_preemptible
+    on_host_maintenance = var.vm_flag_preemptible ? "TERMINATE" : "MIGRATION"
```

Comment:
Same as above re 'MIGRATE' vs 'MIGRATION'

Reply:
Actually it should, damn autocorrect. But the mystery remains in the fact that I tried both and they worked. I’ll change it anyway. Thanks @JarrodBaker !

Reply:
Done, I pushed to main straight away, I'm not sure why GCP was not complaining about it.